concepts/callbacks/about.md

```diff
@@ -1,68 +1,30 @@
 # About
 
-[_Callbacks_ describe the pattern][wiki-callbacks] where a function receives a function as an argument to invoke when it arrives at a condition. The condition may be that its work is done, or that an event has occurred, or that some predicate passes. It can be synchronous; it can be asynchronous.
+[_Callback_ functions][wiki-callbacks] are functions passed as arguments to other functions. The callback function may then be invoked to create a sequence of events. Often, _callbacks_ are used to handle the results of work done, or handle an action when an event occurs. _Callback_ functions can be used in synchronous and asynchronous programming.
 
-This is a useful pattern in JavaScript because it is designed as a single-threaded runtime where only one function call can be executed at a time. During execution, the runtime cannot respond to other events or continue execution until the function has returned. You might have noticed this on websites when they seem to "freeze" or become unresponsive.
+This is a useful pattern in JavaScript because JavaScript is designed as a single-threaded runtime where only one function call can be executed at a time. During execution, the runtime cannot respond to other events or continue execution until the function has returned.
 
-But many API calls (often I/O functions, timers, and event listeners) use an asynchronous mechanism to place the [current function call][mdn-concurrency-stack] on the side until the work is complete. Upon completion, a callback function is placed on the [message queue][mdn-concurrency-queue] so that when the runtime is in between function calls, the messages are then processed by invoking the callback function.
+Many API calls (I/O functions, timers, and event listeners) use an asynchronous mechanism to place the [current function call][mdn-concurrency-stack] on the side until the work is complete. Upon completion, a callback function is placed on the [message queue][mdn-concurrency-queue] so that when the runtime is in between function calls, the messages are then processed by invoking the callback function.
 
-It is also useful when the `callback` (the argument passed in) may not be defined (created) at the calling site. In other words: it may have been passed down from a different place in the program.
+It is also useful to use _callback functions_ because they may reference variables in its closure scope which are unavailable to the function where it is later invoked.
 
-If the `callback` function _returns_ a boolean or boolean-like value, which is intended to be used (as opposed to a throwaway return value), it's called a predicate.
+## Specific examples of callback functions
```

Comment:
Aren't those array methods taught in other concept exercises? Shouldn't it be those exercises' responsibility to build on prior knowledge and point out that those array functions are callbacks in disguise?

Reply:
We (DJ and me) have the guideline that about.md files are not bound by the syllabus progression because they are used for later reference. So if something is a good fit to explain a concept, we would include it in the about.md no matter where it would show up in the concept tree (if at all). My feeling is that the current example categories have some "overweight" on the complex side. That is why I though a simpler category would balance it out. Not a blocker though. Feel free to click resolve if you prefer it as is.

Reply:
Detail added in https://github.com/exercism/javascript/pull/1725/commits/dd155a509988c5619d8e65a19e1f6686ce9a2c75, sub-titled as "Callbacks in disguise"